src/lava/magma/runtime/message_infrastructure/__init__.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+from ctypes import CDLL, RTLD_GLOBAL
+import os
+
+
+def load_library():
+    lib_name = 'libmessage_infrastructure.so'
+    here = os.path.abspath(__file__)
+    lib_path = os.path.join(os.path.dirname(here), lib_name)
+    if (os.path.exists(lib_path)):
+        CDLL(lib_path, mode=RTLD_GLOBAL)
+    else:
+        print("Warn: No library file")
+    extra_lib_folder = os.path.join(os.path.dirname(here), "install", "lib")
+    if (os.path.exists(extra_lib_folder)):
+        extra_libs = os.listdir(extra_lib_folder)
+        for lib in extra_libs:
+            if '.so' in lib and ('idl' not in lib):
+                lib_file = os.path.join(extra_lib_folder, lib)
+                CDLL(lib_file, mode=RTLD_GLOBAL)
+
+
+load_library()
+
+
+from lava.magma.runtime.message_infrastructure. \
+    MessageInfrastructurePywrapper import (  # noqa
+        CppMultiProcessing,
+        ProcessType,
+        Actor,
+        ActorStatus,
+        ActorCmd,
+        RecvPort,
+        AbstractTransferPort,
+        support_grpc_channel,
+        support_fastdds_channel,
+        support_cyclonedds_channel)
+
+from lava.magma.runtime.message_infrastructure.MessageInfrastructurePywrapper \
+    import ChannelType as ChannelBackend  # noqa: E402
+
+from .ports import (  # noqa: E402
```

Comment:
Check if these relative imports cause issues while running tests on lava-loihi.

Reply:
Maybe this needs to change. Joyesh, our vlab environment still not ready for running on Loihi... Last email still has no respond

Reply:
Decision was to use fully qualified imports instead of relative imports.